### ISSUE TITLE
Snap snot bug

### ISSUE THREAD

**Issue by KiltedChro** [xws-bench/bench#106]
Looks like after a ship performs a Snap Shot attack the turn doesn't progress to the next phase.

**Comment by Sleet01** [comment]
Yeah, this is a known issue.  The current implementation is bugged.  There's an earlier issue with the details.
